common/browser_context_options.go

```diff
@@ -232,3 +232,29 @@ func (w *WaitForEventOptions) Parse(ctx context.Context, optsOrPredicate goja.Va
 
 	return nil
 }
+
+// GrantPermissionsOptions are the options used by the browserContext.grantPermissions
+// API.
+type GrantPermissionsOptions struct {
+	Origin string
+}
+
+// NewGrantPermissionsOptions creates a new instance of GrantPermissionsOptions.
+func NewGrantPermissionsOptions() *GrantPermissionsOptions {
+	return &GrantPermissionsOptions{}
+}
+
+// Parse will parse the options. It can parse origin out of the supplied object.
```

Comment:
Also, here since Go doesn't have objects and instances:

```suggestion
// Parse parses the options from opts if opts exists in the Goja runtime.
```